pkgs/development/python-modules/block-io/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, fetchpatch, buildPythonPackage, base58, ecdsa, pycryptodome, requests, six }:
+
+buildPythonPackage rec {
+  pname = "block-io";
+  version = "1.1.8";
+
+  src = fetchFromGitHub {
+    owner  = "BlockIo";
+    repo   = "block_io-python";
+    rev    = "7d532edcfbcc85da68782d4ec0c7767a0008183f";
```

Comment:
Here as well. Also, why don't you use `fetchPypi`?

Reply:
You are right, I fetched from GitHub so I could apply my patch but now that it's included in the latest version I can use `fetchPypi` again.